test/core/end2end/tests/connectivity.cc

```diff
@@ -135,9 +140,9 @@ static void test_connectivity(grpc_end2end_test_config config) {
      READY is reached */
   while (state != GRPC_CHANNEL_READY) {
     grpc_channel_watch_connectivity_state(
-        f.client, state, grpc_timeout_seconds_to_deadline(3), f.cq, tag(4));
+        f.client, state, grpc_timeout_seconds_to_deadline(10), f.cq, tag(4));
```

Comment:
With respect to the flakiness bug we're solving, this change made no difference in my testing.

Reply:
I increased since we increased the timeout. Anything above 5 should be fine here imo